ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/location/dms-textfield.tsx

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
📝  A wrapper class for some pre-existing masked inputs for degrees, minutes, seconds.